SofaKernel/modules/SofaBaseMechanics/MechanicalObject.h

```diff
@@ -422,8 +417,11 @@ protected :
 
     /// @}
 
-    sofa::core::objectmodel::DataFileName filename;
-    Data< bool> ignoreLoader;
+    //int vsize; ///< Number of elements to allocate in vectors
+    Data< int > d_size;
```

Comment:
I don"t see the conflict. If a "size" is given to parse(), then it will be used to call resize(), as was the case before. The only addition is that the Base::parse() will parse it again and put the same value in the Data, but they are the same so it is harmless. The rest of the (broken) logic in init() is unaffected by exposing vsize as a Data instead of a hidden member.